readme.md

```diff
@@ -8,7 +8,7 @@ Thanks to everyone who continues to contribute; new languages are created every
 Make sure to see [contributing.md](/contributing.md) for instructions on contributing to the project!
 
 <!--Languages start-->
-## Languages (846 total)
+## Languages (847 total)
```

Comment:
Since you're removing a file to add another one, the total language count doesn't change.

Reply:
done